### ISSUE TITLE
SVA_MISSING_FAIL_AB & SVA_NO_PASS_AB Violations

### ISSUE THREAD

**Issue by kmnithyame** [svenka3/pyslint#9]
```verilog
module test_new_m;
   bit req,grnt;
   bit clk;

   property p_req_cycle;
           @(posedgeclk) $rose (req) |-> ##[1:3] grnt;
   endproperty : p_req_cycle

   a_new : assert property ((@posedge clk) p_req_cycle);
       else $error ("Fail");


endmodule : test_new_m
```

Log
```log
PySlint: Violation: [SVA_MISSING_FAIL_AB]: Missing FAIL Action block - use $error/`uvm_error:
  // BAD - missing FAIL action block in SVA
  a_var1 : assert property (@(posedge clk) var1);
PySlint: Violation: [SVA_NO_PASS_AB]: Avoid using PASS Action block - likely to cause too many vacuous prints:
  // BAD - avoid using PASS action block in SVA
  // It usually leads to too many vacuous prints
  a_var1 : assert property (@(posedge clk) var1)
    begin : pass_ab
      $info ("Pass Action Block");
    end : pass_ab
   else $error ("Fail AB");
```

**Comment by svenka3** [comment]
Thanks, add a test case and command line please

**Comment by svenka3** [comment]
Your code has syntax errors hence throwing wrong violations. Given the slang/PySlang handling of SV syntax being naive, this is expected. Your corrected code works fine, see below:
```verilog
module test_new_m;
   bit req,grnt;
   bit clk;

   property p_req_cycle;
           // ERROR @(posedgeclk) $rose (req) |-> ##[1:3] grnt;
       @(posedge clk) $rose (req) |-> ##[1:3] grnt;
   endproperty : p_req_cycle

  // ERROR  a_new : assert property ((@posedge clk) p_req_cycle);
   a_new : assert property (@ (posedge clk) p_req_cycle);
       else $error ("Fail");


endmodule : test_new_m
```
